CSVSource/build.gradle

```diff
@@ -35,6 +35,12 @@ applicationDistribution.from("src/main/resources") {
 }
 
 dependencies {
+    // https://mvnrepository.com/artifact/com.sun.net.httpserver/http
+    //implementation group: 'com.sun.net.httpserver', name: 'http', version: '20070405'
+     compile 'com.sun.net.httpserver:http:20070405'
```

Comment:
What is the licensing on this?  Many of the Oracle/Sun things have a license that we generally don't like.  While this is somewhat separate, we'll need to document the licensing here for customers who use this to accept.